File: DemoApp.DataModel/Entities/Person.cs
using System;

namespace DemoApp.DataModel.Entities
{
	public class Person : ContentObject
	{
		public string FirstName { get; set; }
		public string LastName { get; set; }
		public DateTime BirthDate { get; set; }
		public int Gender { get; set; }
	}
}

File: DemoApp.Web.Angular/Scripts/Application/Models/Person.ts
/// <reference path="../_references.ts"/>

module Application {

    export enum Gender {
        Male,
        Female
    }

    export class Person extends Content {
        FirstName: string;
        LastName: string;
        BirthDate: string;
        Gender: Gender;
        constructor(model: Person) {
            super(model);
            this.FirstName = model.FirstName;
            this.LastName = model.LastName;
            this.BirthDate = model.BirthDate;
            this.Gender = model.Gender;
        }
    }
}